tasks/82_bouncing-balls/bouncing-balls.js

```diff
@@ -0,0 +1,9 @@
+export const bouncingBall = ( h, bounce, window ) => {
+  if ( h <= 0 || bounce <= 0 || bounce >= 1 || window >= h ) return -1;
+  let result = 1;
+  while (( h *= bounce ) > window ) {
+    result += 2;
+  }
```

Comment:
I've also solve this as you, but there is a way to solve this without loop:)
`Math.floor(Math.log(w/h)/Math.log(b)) * 2 + 1`

Reply:
done